client/components/OodlerEditTile.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <div class="oodler-edit-tile">
+    <div class="thumbnail">
```

Comment:
I need some kind of container, but I can use `well` to make it more logical. Effect is the same.